phing/tasks/deploy.xml

```diff
@@ -18,6 +18,10 @@
                     promptText="Enter a deploy branch (defaults to [current branch]-build)"
                     promptCharacter=":"
                     defaultValue="${branch.current}-build"/>
+    <propertyprompt propertyName="deploy.tag" useExistingValue="true"
```

Comment:
I think we need to split this into two prompts:


- [ ] Add a prompt to the deploy target which asks Would you like to create a tag?.
- [ ]  If yes, prompt for tag name "Enter the name of the tag to create:" and use input to set the value of deploy.tag property.

Otherwise, this will always create a tag.